build_tools/release_to_github.py

```diff
@@ -138,15 +138,35 @@ def release(config, tag_name, release_sha, s3_release):
     base_url = "https://" + urlparse.urlparse(config.archive_path).hostname
 
     def is_main_file(path):
-        return os.path.basename(path) in ('bob.jar', 'Defold-x86_64-darwin.dmg', 'Defold-x86_64-linux.zip', 'Defold-x86_64-win32.zip')
+        return os.path.basename(path) in ('bob.jar',
+                                          'Defold-x86_64-darwin.dmg',
+                                          'Defold-x86_64-linux.zip',
+                                          'Defold-x86_64-win32.zip')
+
+    def is_platform_file(path):
+        return os.path.basename(path) in ('gdc',
+                                          'gdc.exe')
+
+    def get_platform(path):
+        if 'linux' in path: return 'linux'
+        if 'darwin' in path: return 'macos'
+        if 'macos' in path: return 'macos'
+        if 'win32' in path: return 'win'
+        return ''
+
+    def convert_to_platform_name(path):
+        platform = get_platform(path)
+        basename = os.path.basename(path)
+        name, ext = os.path.splitext(basename)
+        return '%s-%s%s' % (name, platform, ext if ext else '')
```

Comment:
Since the release on github doesn't have folders, we need to modify the filename instead.